konrad/core.py

```diff
@@ -126,6 +129,10 @@ def __init__(
                 log messages are generated. You have to enable the logging
                 by using the :module:`logging` standard library or
                 the `konrad.enable_logging()` convenience function.
+
+            timestep_adjuster (callable): A callable object, that takes the
+                current timestep and the temperature change as input
+                to calculate a new timestep (`f(timestep, deltaT)`).
```

Comment:
I'd add a comment saying that it defaults to keeping a fixed timestep.

Reply:
I will add a sentence.